package.json

```diff
@@ -115,10 +115,10 @@
     "stylelint-config-terra": "^3.0.0",
     "stylelint": "^10.0.1",
     "terra-aggregate-translations": "^1.0.0",
-    "terra-dev-site": "^5.0.0",
```

Comment:
I'll revert this back to `^5.0.0` but wanted to show the accessibility tests passing on travis CI with the new WCAG 2.1 AA rules applied.